components/module-cellery/src/cellery/cellery.bal

```diff
@@ -552,7 +552,13 @@ public function getCellEndpoints(InstanceState[] iNameList, string alias = "", s
 
 public function runDockerTest(string imageName, map<Env> envVars) returns (error?){
     ImageName iName = getCellImage();
-    string instanceName = <string>iName["instanceName"];
+    string? instanceNameResult = iName["instanceName"];
+    string instanceName;
+    if (instanceNameResult is string && instanceNameResult != "") {
+        instanceName = instanceNameResult;
+    } else {
+        instanceName = "tmp";
```

Comment:
Can there be an occasion where two instances are running with the same instanceName 'tmp'? Or else are we sure that there will be always a value for iName["instanceName"] ?

Reply:
We set the name from within the code. So supposedly there will always be a value to instanceName. Since it is an optional attribute, we have handles it this way.